apps/trading/client-pages/markets/closed.tsx

```diff
@@ -216,7 +216,7 @@ const ClosedMarketsDataGrid = ({
         },
       },
       {
-        headerName: t('Mark price'),
```

Comment:
We'll also need to actually render last trade for spots

Reply:
updated